superset-frontend/cypress-base/package.json

```diff
@@ -10,14 +10,15 @@
   "author": "Apcahe",
   "license": "Apache-2.0",
   "dependencies": {
-    "@cypress/code-coverage": "^3.9.2",
+    "@cypress/code-coverage": "^3.9.11",
     "@superset-ui/core": "^0.17.42",
     "react-dom": "^16.13.0",
     "rison": "^0.1.1",
     "shortid": "^2.2.15"
   },
   "devDependencies": {
-    "cypress": "^6.3.0",
+    "@cypress/webpack-dev-server": "^1.6.0",
```

Comment:
curious, what happens if you remove this dependency?

Reply:
I ran all the e2e tests with and without this dependency. There are actually more passing tests without this, so I've removed it.